Assets/MRTK/SDK/Features/UX/Scripts/Pointers/PokePointer.cs

```diff
@@ -434,15 +432,31 @@ bool IMixedRealityNearPointer.TryGetNearGraspPoint(out Vector3 position)
         /// <inheritdoc />
         bool IMixedRealityNearPointer.TryGetDistanceToNearestSurface(out float distance)
         {
-            distance = closestDistance;
-            return true;
+            if (closestProximityTouchable != null)
```

Comment:
Are you seeing issues with the current return values in MRTK code? Or are you calling these yourself and they're causing issues?

The only place I see these two methods called in MRTK code is in FingerCursor, and that script specifically makes sure `nearPointer.IsNearObject` is true before using this data. `IsNearObject` for the PokePointer is this exact `closestProximityTouchable` null check.

Reply:
We were seeing errors logged from this in `FingerCursor` when it did it's orientation calculation. That said, our fork of MRTK is a little behind what's here in `main` and it does look like the this calculation has changed so it's very likely that the issue we were seeing won't occur here.

*However*, it does still seem to me that `TryGetDistanceToNearestSurface()` is part of this class's public interface and can be called at any point, regardless of whether the caller first checks `IsNearObject`.